modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/NotificationsSettingsFragment.kt

```diff
@@ -140,6 +135,13 @@ class NotificationsSettingsFragment :
             )
             true
         }
+        playOverNotificationPreference?.setOnPreferenceChangeListener { _, newValue ->
+            analyticsTracker.track(
+                AnalyticsEvent.SETTINGS_NOTIFICATIONS_PLAY_OVER_NOTIFICATIONS_TOGGLED,
+                mapOf("enabled" to (newValue != "2"))
```

Comment:
One thing to note around this, in the old form the tracking is a boolean since the setting is a switch and is either on or off:
```
     AnalyticsEvent.SETTINGS_NOTIFICATIONS_PLAY_OVER_NOTIFICATIONS_TOGGLED,
     mapOf("enabled" to newValue as Boolean)
```

I assume you're using this behind the scenes to do analytics and so I can't just create a new event, so to keep it useful, I kept the event the same, so true is when the setting is duck or always and false is when the setting is never.

If you want me to improve this in some way please let me know!

Reply:
> we can also add a new property with the form of "value" to "never"/"duck"/"always". wdyt?

Yea that sounds good! Would you like me to add that now in this PR and you guys can add it behind the scenes when you get some time or would you rather me leave it out of this PR and you can add it in on both sides at the same time?

Reply:
Let's include it in this PR.